### DISCUSSION TITLE
plan for GRPC ?

### DISCUSSION THREAD

**Discussion by Ranganaths** [EricLBuehler/mistral.rs#311] [Q&A]
Hi,
 Is there an plan to enable GRPC interface along with REST ? 

thank you,

**Comment by EricLBuehler** [comment]
Hi @Ranganaths! 

I think this would be a valuable addition. Are you able to attempt adding this? If not, I'll try to work on it in the near future.

**Reply by Ranganaths** [reply]
hi @EricLBuehler ,
I am working on it, hopefully should raise PR asap.

thanks

**Reply by EricLBuehler** [reply]
Great, thanks! Please let me know if you need any help.

**Comment by oldgithubman** [comment]
> Hi, Is there an plan to enable GRPC interface along with REST ?
> 
> thank you,

If this leads to distributed inference, thumbs up!
